### PR TITLE
feat: (sfui2-909) display vertical block

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes[ #909](https://vsf.atlassian.net/browse/SFUI2-909)

# Scope of work

Vertical banner added in two versions two and four in the row on desktop. 

# Screenshots of visual changes

Screenshot from 2023-04-03 16-20-27

Multiple variant:

Screenshot from 2023-04-03 16-20-43

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/docs/components/blocks/Banners.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: A breadcrumb trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place.
```
Comment (Szymon-dziewonski) [inline apps/docs/components/blocks/Banners.md:4]: I think that description is from original file this one was copied from

File: apps/preview/nuxt/pages/showcases/Banners/VerticalBanner.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="flex flex-col gap-6 md:flex-row">
+    <div class="flex flex-col justify-between rounded-md md:items-center md:basis-1/2 bg-warning-200">
+      <div class="flex flex-col items-center p-4 text-center md:p-10">
```
Comment (Szymon-dziewonski) [inline apps/preview/nuxt/pages/showcases/Banners/VerticalBanner.vue:4]: question: if we have `items-center` do we need `text-center` ?

File: apps/docs/components/blocks/Banners.md (hunk 1 above)
Comment (AdamPawlinski) [reply apps/docs/components/blocks/Banners.md:4]: Thx, just forgot about changing.

File: apps/preview/nuxt/pages/showcases/Banners/VerticalBanner.vue (hunk 2 above)
Comment (AdamPawlinski) [reply apps/preview/nuxt/pages/showcases/Banners/VerticalBanner.vue]: It is still needed to center text inside the paragraphs.

**Review by Szymon-dziewonski — APPROVED** [review]

**Review by justyna-13 — COMMENTED** [review]
I'm wondering about focus - were there any changes or it should be like on figma and whole display should also be focused?

**Comment by AdamPawlinski** [comment]
@justyna-13 Thx, you are absolutely right, Changed it.

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**